environments/prod/special-tribunals-service-gov-uk.yml

```diff
@@ -20,3 +20,14 @@ cname:
     ttl: 300
     record: "afdverify.hmcts-prod-hdgpbqdkafhmcse9.z01.azurefd.net"
     shutter: false
+  - name: "update"
```

Comment:
Not same as your test, https://github.com/hmcts/azure-public-dns/commit/967c8cee762fead8f02fd18ea8007e39f0c6827e

are you looking the prod file as below?